README.md

```diff
@@ -8645,6 +8647,29 @@ There is an exception, though: *experimental features*, marked with the "**:fire
 Those experimental features are not protected by semantic versioning, and may break between two minor releases of the library. To help them becoming stable, [your feedback](https://github.com/groue/GRDB.swift/issues) is greatly appreciated.
 
 
+### SQLite error 21 "wrong number of statement arguments" with LIKE queries
+
+You may get the error "wrong number of statement arguments" when executing a LIKE query similar to:
+
+```swift
+let name = textField.text
+try dbQueue.read { db in
+    try db.execute(sql: "SELECT * FROM students where title like '%?%'", arguments: [name])
+}
+```
+
+What is important to understand here is that SQLite only interprets `?` as a parameter when it is a placeholder for a whole value (int, double, string, blob, null). In this incorrect query, `?` is just a character in the `'%?%'` string value: it is not a query parameter, and is not processed in any way. See [https://www.sqlite.org/lang_expr.html#varparam](https://www.sqlite.org/lang_expr.html#varparam) for more information about SQLite parameters.
+
+To provide the parameter, one option is to let SQLite build the like pattern using the string concatenation operator `||`:
```

Comment:
Since you provide a single example of code that works, then please provide the one that lets Swift build the parameter instead:

```swift
let name = textField.text
let players = try dbQueue.read { db in
    let pattern = "%\(name)%"
    try Player.fetchAll(db, sql: "SELECT * FROM player WHERE name LIKE ?", arguments: [pattern])
}
```

Why so? Because it can trivially be ported to the query interface when the user decides to use it:

```swift
let name = textField.text
let players = try dbQueue.read { db in
    let pattern = "%\(name)%"
    try Player.filter(Column("name").like(pattern)).fetchAll(db)
}
```

You don't have to mention the query interface request in your FAQ. But we have to write documentation in a way that smoothes the transition from raw SQL to generated SQL.